Misc/NEWS.d/next/Core_and_Builtins/2024-12-13-14-17-24.gh-issue-127903.vemHSl.rst

```diff
@@ -0,0 +1,2 @@
+Fix segmentation fault in debug mode in ``_copy_characters`` when there is
+nothing to copy.
```

Comment:
On one's hand, it could be ok to say that we fixed a crash when copying characters does nothing, but on the other hand, a clear formulation of what we did is a bit hard to find. We don't want to mention `_copy_characters` which is entirely private and saying that we fix a crash when there is nothing to copy seems unhelpful for users. So I'd suggest removing the NEWS.

Reply:
I agree that it is difficult to find a clear definition of what we have done. However, I would prefer to leave this news entry as it is, because some users (in 3.12 and 3.13) may encounter this issue and they will probably open the gdb and see the stack trace, which leads to the ``_copy_characters`` function, and then search the changelog for anything that mentions this function.

Reply:
Ok for keeping the function name but maybe add that it's in `unicodeobject.c`. Something like:

```rst
``Objects/unicodeobject.c``: fix a crash on DEBUG builds in ``_copy_characters``
when there is nothing to copy.
```

Reply:
Agreed. This looks much better.